src/scss/components/_threeImages.scss

```diff
@@ -0,0 +1,45 @@
+.images{
+    margin: 60px 0;
```

Comment:
Please go to this page: http://localhost:8080/restauracion.html and check the spacing around the component. As I see it, you can remove the margin from the `.images` class, and only set a `margin-top: 60px;` in `md` viewport